pandas/tests/frame/test_duplicates.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import numpy as np
+from pandas import Series, DataFrame
+
+from pandas.util.testing import assert_series_equal, assert_frame_equal
+import pandas.util.testing as tm
+
+
+class TestDataFrameDuplicated(object):
```

Comment:
I think these tests should be in `tests/frame/test_analytics.py`.  It looks like there are a couple bug `DataFrame.duplicated` tests there, though nothing expansive, e.g.

https://github.com/pandas-dev/pandas/blob/4c4579d5629d7aed1719dbc8aaa423681b343772/pandas/tests/frame/test_analytics.py#L1634-L1649

The `Series.duplicated` tests are also in the associated `series/tests_analytics.py` file, so makes sense to mirror that setup.